src/database/mod.rs

```diff
@@ -29,6 +30,8 @@ pub struct Database {
     pub pool: Pool<Postgres>,
 }
 
+pub type SharedDatabase = Arc<Database>;
```

Comment:
`Arc<Database>` is actually fewer characters than `SharedDatabase` and obfuscates less